src/algebra/algebra/basic.lean

```diff
@@ -1002,6 +1002,12 @@ lemma of_alg_hom_symm (f : A₁ →ₐ[R] A₂) (g : A₂ →ₐ[R] A₁) (h₁
 noncomputable def of_bijective (f : A₁ →ₐ[R] A₂) (hf : function.bijective f) : A₁ ≃ₐ[R] A₂ :=
 { .. ring_equiv.of_bijective (f : A₁ →+* A₂) hf, .. f }
 
+@[simp] lemma coe_of_bijective {f : A₁ →ₐ[R] A₂} {hf : function.bijective f} :
+  (alg_equiv.of_bijective f hf : A₁ → A₂) = f := rfl
+
+@[simp] lemma of_bijective_apply {f : A₁ →ₐ[R] A₂} {hf : function.bijective f} (a : A₁) :
+  (alg_equiv.of_bijective f hf) a = f a := rfl
```

Comment:
I kind of expect the linter to complain about having both of these...

Also, can you get `simps` to generate the second one?

Reply:
Ahh, yes I forgot about `simps {simp_rhs}`. But I think doing it manually is better, actually: the `ring_equiv` version only has the `coe` version as `@[simp]`.